src/shared_modules/content_manager/doc/components/PUB_SUB_PUBLISHER.md

```diff
@@ -0,0 +1,66 @@
+# Pub Sub Publisher stage
+
+## Details
+
+The [pub-sub publisher](../../src/components/pubSubPublisher.hpp) stage is part of the Content Manager orchestration and is in charge of publishing the Content Updater content data that will be then read by the consumers.
+
+> Note: The `pub-sub` preffix makes reference to the [Publish-Subscribe pattern](https://github.com/wazuh/wazuh/issues/16786).
+
+The content data includes:
+- Content file paths: A list of paths of the files that hold the downloaded content from previous stages.
+- Content Updater stages status: A list that summarizes which stages where executed in the orchestration along with their status (whether `ok` of `fail`).
+
+It's important to note that if the list of paths is empty, meaning that there is no new content to process,  the publishing is skipped.
+
+The publication is made throw a [router provider](../../../router/include/iRouterProvider.hpp) and into a channel, so that the channel subscribers will be able to consume the published content.
```

Comment:
I don't understand the first part. Could it be?
```suggestion
The publication is made through a [router provider](../../../router/include/iRouterProvider.hpp) and into a channel, so that the channel subscribers will be able to consume the published content.
```